ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -215,7 +215,7 @@ - (RTCVideoTrack *)createVideoTrack:(NSDictionary *)constraints {
 RCT_EXPORT_METHOD(mediaStreamTrackSetEnabled:(nonnull NSString *)trackID : (BOOL)enabled)
 {
   RTCMediaStreamTrack *track = [self trackForId:trackID];
-  if (track && track.isEnabled != enabled) {
```

Comment:
why is the check unneeded? Is that checked on another level? Otherwise it would be starting the capturer twice?

Reply:
It's checked in JS already. Since we are now waiting for the capture to start, there is no way for 2 starts to happen in parallel. Calling start twice is ok, for example to change the resolution.

Reply:
I think it would be god to put that explanation in the commit message.

Reply:
Can do!